docs/guide/custom_policies.md

```diff
@@ -94,26 +94,27 @@ It is not possible to remove an existing module, function or limitation from a P
 
 ## Integrating the `PolicyProvider` into EzPublishCoreBundle
 
-For a `PolicyProvider` to be active, it must be properly declared in EzPublishCoreBundle.
-A bundle just has to retrieve CoreBundle's DIC extension and call `addPolicyProvider()`. This must be done in the bundle's `build()` method.
+For a `PolicyProvider` to be active, you have to register it in the class `src/Kernel.php`:
 
-``` php
-namespace Acme\ExampleBundle\AcmeExampleBundle;
+```php
+namespace App;
 
-use Symfony\Component\HttpKernel\Bundle\Bundle;
+use App\Security\AppPolicyProvider;
+use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\HttpKernel\Kernel as BaseKernel;
 
-class AcmeExampleBundle extends Bundle
+class Kernel extends BaseKernel
 {
-    public function build(ContainerBuilder $container)
-    {
-        parent::build($container);
+    use MicroKernelTrait;
 
+    protected function configureContainer(ContainerBuilder $container, LoaderInterface $loader): void
+    {
         // ...
- 
+        
         // Retrieve "ezpublish" container extension
         $eZExtension = $container->getExtension('ezpublish');
         // Add the policy provider
-        $eZExtension->addPolicyProvider(new MyPolicyProvider());
+        $eZExtension->addPolicyProvider(new AppPolicyProvider());
```

Comment:
It looks like now it's inconsistent, L43 & L69 still use "MyPolicyProvider".

Reply:
@micszo done!